includes/definitions/discovery/apc-epdu.yaml

```diff
@@ -0,0 +1,36 @@
+mib: PowerNet-MIB
+modules:
+    os:
+        hardware:
```

Comment:
This doesn't need to be an array.  Don't these need the mib in the oid to work?
PowerNet-MIB::ePDUDeviceStatusModelNumber

Reply:
my bad, i fixed it